index.js

```diff
@@ -5,7 +5,36 @@ var mat2d = glMatrix.mat2d;
 var vec2 = glMatrix.vec2;
 
 function Context(width, height) {
+  if (width == null) {
+    width = process.stdout.columns * 2 - 2;
+  }
+  if (height == null) {
+    height = process.stdout.rows * 4;
+  }
+
   this._canvas = new Canvas(width, height);
+
+  Object.defineProperties(this, {
```

Comment:
The only way to provide dynamic properties compatible with browser. Works in node@0.10